src/locale/locales/es/messages.po

```diff
@@ -6717,7 +6717,7 @@ msgstr "Mostrar advertencia y filtrar de los feeds"
 
 #: src/view/com/post-thread/PostThreadFollowBtn.tsx:128
 #~ msgid "Shows posts from {0} in your feed"
-#~ msgstr ""
+#~ msgstr "Mostrar los posts de {0} en tu feed"
```

Comment:
```suggestion
#~ msgstr "Mostrar las publicaciones de {0} en tu feed"
```

Reply:
I just saw these! maybe we you include them in another PR? 🙏🏽